### ISSUE TITLE
Spicetify Custom App react symbol error

### ISSUE THREAD

**Issue by sebigut** [khanhas/spicetify-cli#1029]
## ℹ Computer information

- Spotify version: Spotify for Windows 1.1.67.586.gbb5ef64e
- Spicetify version: 2.6.1

## 📝 Provide detailed reproduction steps (if any)

1. Try to apply spicetify after upgrading to this Spotify version
2. errors with custom app react symbols

### ✔️ Expected result

_What is the expected result of the above steps?_

Spicetify to apply

### ❌ Actual result

_What is the actual result of the above steps?_

Errors applying spicetify. To note, Spotify doesnt restart and Im still able to use it, but wont apply the config

## 📷 Screenshots

_Are there any useful screenshots?_
![imagen](https://user-images.githubusercontent.com/14024884/131847716-adb08b95-1ec4-463c-aaff-8a55ef6d5a50.png)

Spicetify config state:
![imagen](https://user-images.githubusercontent.com/14024884/131848171-fb62c1e2-ec2a-4599-9b37-f3a7dc046ca7.png)

**Comment by sebigut** [comment]
Update: after removing the custom apps spicetify applies succesfully, but still gives this:

Applying additional modifications:
error Cannot find symbol for Custom app React symbols

**Comment by RobzHD** [comment]
I'm still having the same issue

**Comment by braandoned** [comment]
same issue here

**Comment by kegobass** [comment]
I can also confirm I get the same error on 2.6.1 and the latest build of Spotify.

**Comment by CharlieS1103** [comment]
Did you download spotify via a package manager?

**Comment by applekillerbyjames** [comment]
don't know why they closed #845 , still broken with same issue

**Comment by CharlieS1103** [comment]
Try running ```spicetify restore backup apply```

**Comment by stankovictab** [comment]
@CharlieS1103 I'm on Windows running the version from the website, 
still with `spicetify restore backup apply` I'm getting the same error.

**Comment by CharlieS1103** [comment]
Mind trying a full reinstall of spotify and spicetify?

**Comment by stankovictab** [comment]
Spotify version 1.1.67.586.gbb5ef64e, reinstalled Spicetify too, on 2.6.1, theme changing works but extensions and custom apps don't.

**Comment by CharlieS1103** [comment]
I'm confused, so you no longer have the react error?

**Comment by stankovictab** [comment]
Let me rephrase, after reinstalling I can change the theme without editing the custom_apps field, but if I make some changes to the config file, say change to another theme, and then change the custom_apps entry, I get the goroutine error and nothing would apply.

**Comment by CharlieS1103** [comment]
Try just not running the reddit app, pretty sure it's broken(not sure tho.)

**Comment by stankovictab** [comment]
Nope, nothing to do with the reddit app, didn't include it in the first place.

**Comment by nickclyde** [comment]
Just did a fresh install of the new Spotify and spicetify 2.6.2 from AUR. Am able to apply themes just fine, but if I try to apply any of the custom apps I get the same:
```
❯ spicetify apply
spicetify vDev
Overwriting themed assets:
OK
Transferring user.css:
OK
Applying additional modifications:
error Cannot find symbol for Custom app React symbols
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/khanhas/spicetify-cli/src/apply.insertCustomApp.func1({0xc0003e8000, 0x16686f})
	/home/nick/.cache/yay/spicetify-cli-git/src/spicetify-cli/src/apply/apply.go:180 +0x99b
github.com/khanhas/spicetify-cli/src/utils.ModifyFile({0xc0000a05c0, 0x1e}, 0xc0001239e8)
	/home/nick/.cache/yay/spicetify-cli-git/src/spicetify-cli/src/utils/utils.go:190 +0x82
github.com/khanhas/spicetify-cli/src/apply.insertCustomApp({0xc0000a05c0, 0x21}, {{0x950058, 0x0, 0x0}, {0xc0000aa200, 0x1, 0x2}, 0x1, 0x1})
	/home/nick/.cache/yay/spicetify-cli-git/src/spicetify-cli/src/apply/apply.go:155 +0x85
github.com/khanhas/spicetify-cli/src/apply.AdditionalOptions({0xc00001a330, 0x11}, {{0x950058, 0x0, 0x0}, {0xc0000aa200, 0x1, 0x2}, 0x1, 0x1})
	/home/nick/.cache/yay/spicetify-cli-git/src/spicetify-cli/src/apply/apply.go:34 +0x3df
github.com/khanhas/spicetify-cli/src/cmd.Apply({0x6e13e0, 0x3})
	/home/nick/.cache/yay/spicetify-cli-git/src/spicetify-cli/src/cmd/apply.go:73 +0x905
main.main()
	/home/nick/.cache/yay/spicetify-cli-git/src/spicetify-cli/spicetify.go:192 +0x4ac
```

If I remove the custom apps from the config, `spicetify apply` runs successfully

**Comment by afonsojramos** [comment]
@nickclyde hi! Soooo, yeah, this is because Spotify completely changed the class names, which is forcing us to rebuild the CSS map almost from scratch. There are already some custom apps that work, however, some do not. Best I can advise you is to either run an older version of Spotify with an accompanying version of spicetify.

**Comment by afonsojramos** [comment]
@nickclyde however your version shows vDev, did you install from master?

**Comment by DarkVillager** [comment]
same issue here without any of the custom apps

**Comment by cloudaerow** [comment]
> @nickclyde hi! Soooo, yeah, this is because Spotify completely changed the class names, which is forcing us to rebuild the CSS map almost from scratch. There are already some custom apps that work, however, some do not. Best I can advise you is to either run an older version of Spotify with an accompanying version of spicetify.

how do i download a older version of spicetify?

**Comment by itsmeow** [comment]
I've solved this locally, but there are still several hooks broken. Spotify needs to stop changing their stuff...

This isn't a CSS map issue, not this part at least.

**Comment by nickclyde** [comment]
@afonsojramos Yea, I am running [spicetify-cli-git](https://aur.archlinux.org/packages/spicetify-cli-git/) from AUR which is based on `master`. Sounds like there is a fix in the works so I'll keep an eye out for that, thanks for all the hard work!

**Comment by swanstriker47** [comment]
Just manually uninstall Spotify and then run this command in PowerShell
iwr -useb https://raw.githubusercontent.com/spicetify/spicetify-cli/master/install.ps1 | iex
iwr -useb https://raw.githubusercontent.com/spicetify/spicetify-marketplace/master/install.ps1 | iex
